src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -618,7 +619,6 @@ OCC::CfApiWrapper::PlaceHolderInfo OCC::CfApiWrapper::findPlaceholderInfo(const
     if (result == S_OK) {
         return info;
     } else {
-        qCWarning(lcCfApiWrapper()) << "Couldn't get placeholder info" << QString::fromWCharArray(_com_error(result).ErrorMessage());
```

Comment:
@mgallien Why remove this? I thought we are still in the process of improving the VFS and fixing issues. These logs might be useful.

Reply:
many time `OCC::CfApiWrapper::findPlaceholderInfo` is called to see if the info exists and the result is then check by the caller that will also produce a log output
we then gets two log lines or we get this one but the case is handled by the code
having `result != S_OK` is not an error case and the code is expecting it
so the log is being produced but just create noise